samples/tableadmin.js

```diff
@@ -0,0 +1,289 @@
+/**
+ * Copyright 2018, Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+// Imports the Google Cloud client library
+const Bigtable = require('@google-cloud/bigtable');
+
+async function runTableOperations(instanceName, tableName) {
+    // const instanceName = "my-instance";
+    // const tableName = "my-bigtable-name";
+    const bigtableClient = new Bigtable();
+    const instance = bigtableClient.instance(instanceName);
+    const table = instance.table(tableName);
+
+    // Check if table exists
+    console.log('\nChecking if table exists...');
```

Comment:
I would throw the entire tableExistss and creation in a single try catch and then you also won't need to put rescope the variable via the `let tableExists` above